api_app/models/domain/restricted_resource.py

```diff
@@ -0,0 +1,28 @@
+from typing import Optional
+from pydantic import Field
+from models.domain.resource import ResourceType
+from models.domain.azuretremodel import AzureTREModel
+
+
+class RestrictedProperties(AzureTREModel):
+    display_name: str = ""
+    description: str = ""
+    overview: str = ""
+
+
+class RestrictedResource(AzureTREModel):
```

Comment:
Out of curiousity, what were the fields in shared service that we want masked?

Reply:
anything in the properties object that's NOT name/description basically - which would include things like firewall rules / certificate names etc potentially